client/components/UI/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import FormItem from '../FormItem';
+// import styles from './Dropdown.module.scss';
+import { default as ReactDropdown, Group, Option } from 'react-dropdown';
+
+interface DropdownProps {
+    label: string;
+    id: string;
+    name: string;
+    options: (Group | Option | string)[];
+    default: Option | string;
+    placeholder: string;
+    required: boolean;
+}
+
+export default function Dropdown(props: DropdownProps) {
+    return (
+        <FormItem id={props.id} label={props.label} required={props.required}>
+            <ReactDropdown
+                options={props.options}
+                value={props.default}
+                placeholder={props.placeholder}
+            />
+            {/* <select className={styles.dropdownSelect} id={props.id}>
```

Comment:
Try not to commit commented out code, if it's necessary then leave a comment as to why it's commented out

Reply:
This code was not meant to be committed with this branch, I’ll remove it from here and make the changes in the proper branch